esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2684,86 +2683,6 @@ bool CFileSprayEx::onDFUWUFile(IEspContext &context, IEspDFUWUFileRequest &req,
     return true;
 }
 
-int CFileSprayEx::doFileCheck(const char* mask, const char* netaddr, const char* osStr, const char* path)
```

Comment:
It contains two parts.  The second part is replaced by: checkDropZoneIPAndPath(). The first part is only used in one location. The part is moved to line 2727-2732.